### PR TITLE
Follow grad_norm changes in transformers #27326

### PR DESCRIPTION
**Pull request by jingyanwangms**
# What does this PR do?
See issue description in #1723 
It's caused by transformers PR https://github.com/huggingface/transformers/commit/4f09d0fd888dbf2660313f9715992822acfb99ce
Follow the same changes in ORTTrainer

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Review by JingyaHuang — COMMENTED** [review]
Hi @jingyanwangms, thanks for fixing it! 

Just a small typo, and could you also run `make style` to pass the code quality check? Thx!

File: optimum/onnxruntime/trainer.py [hunk 1]
```diff
@@ -744,19 +744,27 @@ def get_dataloader_sampler(dataloader):
                         # deepspeed does its own clipping
 
                         if is_sagemaker_mp_enabled() and args.fp16:
-                            self.optimizer.clip_master_grads(args.max_grad_norm)
+                            _grad_norm = self.optimizer.clip_master_grads(args.max_grad_norm)
                         elif hasattr(self.optimizer, "clip_grad_norm"):
                             # Some optimizers (like the sharded optimizer) have a specific way to do gradient clipping
-                            self.optimizer.clip_grad_norm(args.max_grad_norm)
+                            _grad_norm = elf.optimizer.clip_grad_norm(args.max_grad_norm)
```
[the remark this answers is not in the document]
Comment (jingyanwangms) [reply optimum/onnxruntime/trainer.py]: Sorry accident edit. Fixed now
`make style` changed 10 other files. Should I keep those in this PR to pass the CI?

**Comment by JingyaHuang** [comment]
Hi @jingyanwangms, what black / ruff version did you use? It seems that the code quality CI still failed.

Optimum pinned styling tools to some specific version:
https://github.com/huggingface/optimum/blob/bb21ae7f7d572805f6ecdea8e0f02dc6014d57e8/setup.py#L45

Could you rerun `make style` with these versions and submit all changed files (even if no code change is related to the PR)?

**Review by JingyaHuang — APPROVED** [review]
LGTM, thanks again for the fix!

**Pull request merged**